schema/schema_template.py

```diff
@@ -338,4 +338,38 @@ def validate_comments(self, key, value):
         check_string_length(value, DESCRIPTION_STRING_LENGTH, key)
         return value
 
+class Measurement(Base):
+    # Template table. This is a placeholder for a table that would store measurements such as parallax.
+
+    __tablename__ = 'Measurement'
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+
+    # units should be replaced with the unit of measurement.
+    # e.g., for parallax measured in milliarcseconds, this would read parallax_mas
+    measurement_units = Column(Float, nullable=False)
```

Comment:
At a glance, this made it sound like we are storing a column that has the units which is not what's intended.
I would just call the column `measurement` or `value` and in the comment say that you can specify the unit in the column name (which personally I don't like but I know we've adopted elsewhere).

Reply:
got it!